packages/client/hmi-client/src/components/workflow/ops/simulate-ensemble-ciemss/tera-simulate-ensemble-ciemss-drilldown.vue

```diff
@@ -165,6 +165,10 @@
 											:options="[CiemssMethodOptions.dopri5, CiemssMethodOptions.euler]"
 										/>
 									</div>
+									<div v-if="knobs.method === CiemssMethodOptions.euler" class="label-and-input">
+										<label for="num-steps">Solver step size</label>
+										<tera-input-number inputId="integeronly" v-model="knobs.stepSize" />
```

Comment:
- This `<label for=".." ...`  will throw a warning if there is no target, I would probably just remove the `for` attribute
- I don't know what `inputId` does, but there is nothing saying step-size cannot be fractional

Reply:
eg this thing
<img width="1049" alt="image" src="https://github.com/user-attachments/assets/da453b37-9d98-4e14-bea0-c55c2c951911">

Reply:
Thank you